docs/www/docs.html

```diff
@@ -58,26 +58,27 @@
     <h3>Translations</h3>
     <ul class="list-unstyled">
         <li>
```

Comment:
I see the comment.  Last Update time.  That's not obvious (or terribly useful.)  Alphabetical is going to be the simplest and most obvious.